src/client/visualizations/table/body/measures/measure-row.tsx

```diff
@@ -50,7 +49,6 @@ export const MeasureRow: React.SFC<MeasureRowProps> = props => {
         highlight={highlight}
         scale={scales[i]}
         cellWidth={cellWidth}
-        lastLevel={datum["__nest"] === splitLength}
```

Comment:
What is that, regression?

Reply:
No, this logic is extracted higher in component tree and better named.